File: fixtures/scrapy_output/get_long_form_header.py
from scrapy import Request, Spider


class MySpider(Spider):
    name = 'myspider'

    def start_requests(self):
        headers = {
            'Accept': 'application/json',
            'user-token': '75d7ce4350c7d6239347bf23d3a3e668',
        }
        yield Request(
            'http://localhost:8080/api/retail/books/list',
            callback=self.parse,
            headers=headers,
        )

    def parse(self, response):
        pass

File: test/fixtures/r-httr2/get_long_form_header.r
library(httr2)

request("http://localhost:28139/api/retail/books/list") |>
  req_headers(
    Accept = "application/json",
    `user-token` = "75d7ce4350c7d6239347bf23d3a3e668"
  ) |>
  req_perform()
